AiTableTopGameMaster.Tests/IntegrationTests.cs

```diff
@@ -0,0 +1,88 @@
+using AiTableTopGameMaster.ConsoleApp.Clients;
+using AiTableTopGameMaster.Core.Services;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Moq;
+using Shouldly;
+using Spectre.Console.Testing;
+using Xunit;
+
+namespace AiTableTopGameMaster.Tests;
+
+public class IntegrationTests
+{
+    [Fact]
+    public async Task ConsoleChatClient_WithOutputReviewer_ShouldNotRequireOllamaForBasicFlow()
+    {
+        // Arrange
+        Mock<Agent> mockGameMasterAgent = new Mock<Agent>();
+        Mock<IOutputReviewer> mockOutputReviewer = new Mock<IOutputReviewer>();
+        TestConsole console = new TestConsole();
+        
+        // Setup the output reviewer to always accept the output
+        mockOutputReviewer
+            .Setup(x => x.ReviewOutputAsync(It.IsAny<string>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(OutputReviewResult.Acceptable());
+        
+        ConsoleChatClient client = new(
+            mockGameMasterAgent.Object, 
+            mockOutputReviewer.Object, 
+            console, 
+            NullLogger<ConsoleChatClient>.Instance);
+        
+        // Act & Assert - Just verify the client was created successfully
+        client.ShouldNotBeNull();
```

Comment:
I feel like this isn't testing enough. We don't seem to have a lot of tests that test the actual chat interactions. When I ran this pull request I always got empty string responses back from the system.